### PR TITLE
Litvischenko yevhen/hw5

### PR DESCRIPTION
**Pull request by litvischenkoev**
HW5 is done

### CODE REVIEW

File: LitvischenkoYevhen/HW5/HW5_task_1.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#use loop
```
Comment (kolyasalubov) [inline LitvischenkoYevhen/HW5/HW5_task_1.py:1]: good job

File: LitvischenkoYevhen/HW5/HW5_task_2.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Print Fibnacci numbers
+num = int(input('Enter the number: '))
```
Comment (kolyasalubov) [inline LitvischenkoYevhen/HW5/HW5_task_2.py:2]: you should use meaningful variable names

File: LitvischenkoYevhen/HW5/HW5_task_3.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# calculate factorial
+num = int(input('Enter the number: '))
```
Comment (kolyasalubov) [inline LitvischenkoYevhen/HW5/HW5_task_3.py:2]: you should use meaningful variable names

File: LitvischenkoYevhen/HW5/HW5_task_2.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+fi = [0, 1]
+count = 2
+while count <= num:
```
Comment (kolyasalubov) [inline LitvischenkoYevhen/HW5/HW5_task_2.py:5]: good job

File: LitvischenkoYevhen/HW5/HW5_task_3.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+sum = 1
+for i in range(1,num+1):
```
Comment (kolyasalubov) [inline LitvischenkoYevhen/HW5/HW5_task_3.py:4]: good job
but look at this version of code
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)

**Review by kolyasalubov — APPROVED** [review]

File: LitvischenkoYevhen/HW5/HW5_task_2.py (hunk 2 above)
Comment (litvischenkoev) [reply LitvischenkoYevhen/HW5/HW5_task_2.py:2]: Thank you/ I will try )

**Review by kolyasalubov — APPROVED** [review]

**Pull request merged**